src/lib/components/index/fragments/devx-pod.svelte

```diff
@@ -3,9 +3,11 @@
 	import GradientContentCard from '../../library/gradient-content-card.svelte';
 	import { devXPod } from '$lib/contents/index/devx-content';
 	import Listen from '../../svgs/listen.svelte';
+	import Dot from '../../library/dot.svelte';
 </script>
 
-<ContentSection id="devx-pod" title="DevX Pod" annotation="listen" icon={Listen}>
+<ContentSection class="relative" id="devx-pod" title="DevX Pod" annotation="listen" icon={Listen}>
+	<Dot class="absolute block h-[650px] w-[650px] -translate-y-24 translate-x-[110%]" />
```

Comment:
Hey @nisarhassan12 I've added the big dots, two things need to be fixed though: 
1. The overflow should be hidden I guess
2. It causes issues on mobile devices and responsivness

Do you know a fix for them?